src/NonCumulative/Soundness/Properties/LogRel.agda

```diff
@@ -49,10 +54,17 @@ Glu-wellfounded-≡-Glul : ∀ {j k} →
   (λ {l} l<k → Glu-wellfounded (j + k) {l} (Li≤ refl l<k)) ≡ Glu-wellfounded k
 Glu-wellfounded-≡-Glul {j} {k} = implicit-extensionality fext (fext (λ l<k → Glu-wellfounded-≡′ (Li≤ {j + k} refl l<k) l<k))
 
-Glu-wellfounded-≡-GluΛ : ∀ {j k} →
+Glu-wellfounded-≡-GluΛI : ∀ {j k} →
```

Comment:
why do you need all these specialized cases? abstract them into one common lemma, c.f. https://github.com/HuStmpHrrr/mech-type-theories/blob/db45fb402c6aab78a9bf554a0618655a11e64e9d/src/NonCumulative/Semantics/Properties/PER/Core.agda#L34